_sass/custom/custom.scss

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
similar question with the js includes - is this css included automatically or should it be pulled in via frontmatter (maybe the "overrides" folder was magic?)

Reply:
All of the imports from made by Just the Docs are in [`assets/css/just-the-docs.scss`](https://github.com/pmarsceill/just-the-docs/blob/master/assets/css/just-the-docs.scss). They include both `_sass/overrides` and `_sass/custom/custom` for people who want to modify the theme.

I mentioned in the old version of this thread that I think that in the long term it might make sense for us to move to using `scss_files` in the front matter akin to how we're using `js_files`, but that will be slightly more complicated.